etl/steps/data/garden/emissions/2023-07-10/owid_co2.meta.yml

```diff
@@ -1,12 +1,9 @@
 dataset:
-  title: OWID CO2 dataset (2023)
+  title: OWID CO2 dataset (2023b)
```

Comment:
It's outside the remit of this review, but there is a redundant `DATASET_TITLE = "CO2 dataset (OWID, 2022)"` in `etl/steps/data/garden/emissions/2023-07-10/owid_co2.py`.